src/osdc/Objecter.h

```diff
@@ -28,6 +28,8 @@
 
 #include <boost/container/small_vector.hpp>
 #include <boost/asio.hpp>
+#include <boost/thread/shared_mutex.hpp>
```

Comment:
Nit: I don't see these being used in this header.

Reply:
Right, looks like it's no longer needed.